README.md

```diff
@@ -132,6 +134,24 @@ python run.py --rhbuild 3.2 --global-conf conf/luminous/upgrades/upgrade.yaml --
 --inventory conf/inventory/rhel-7.8-server-x86_64-released.yaml --suite suites/luminous/upgrades/upgrades_containerized.yaml
 --log-level info --ignore-latest-container --insecure-registry --skip-version-compare
 ```
+Store argument usage: 
```

Comment:
Can this be moved under usage section with generic CLI usage just to make it more readable.

Reply:
incorporated the changes